backend/endpoints/rooms.py

```diff
@@ -23,7 +23,7 @@ async def extra_room_info(room_data: dict):
             "group_name": "channel",
             "room_name": room_data["room_name"],
         }
-    return {"category": "direct messagine", "group_name": "dm", "room_name": None}
+    return {"category": "direct messaging", "group_name": "dm", "room_name": None}
```

Comment:
What's group name ?
Is it different from category ?

Reply:
Not sure exactly what role it is supposed to play. But it is one of the fields required in the sidebar.
I think it's more like the plugin name.

Reply:
Please, what if we skip `room_name`for dm. @AI-fae 
```suggestion
    return {"category": "direct messaging", "group_name": "dm"}
```

Reply:
The `extra_room_info `function needs a more complete docstring.

Reply:
I have made a little improvement to the extra_room_info.

Reply:
Seen. Thanks.